### PR TITLE
Add Arsheel Sheikh

### PR DESCRIPTION
**Pull request by arsheelsheikh**

### CODE REVIEW

File: public/data/arsheelsheikh.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Arsheel Sheikh",
+    "bio": "Currently pursing Bachelor of Engineering in CSE from SVPCET, I am tech and open source enthusiast",
```
Comment (Panquesito7) [inline public/data/arsheelsheikh.json:3]:
```suggestion
    "bio": "Currently pursuing Bachelor of Engineering in CSE from SVPCET || Tech and open source enthusiast",
```

File: public/data/arsheelsheikh.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        "url": "https://github.com/arsheelsheikh",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/arsheeltwtse",
+        "icon": "twitter"
+      }
+    ]
+  }
```
Comment (Panquesito7) [inline public/data/arsheelsheikh.json:17]: Missing an endline.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by arsheelsheikh** [comment]
Thanks for the review @Panquesito7

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**